src/redux/configureStore.js

```diff
@@ -82,11 +82,52 @@ const configureStore = () => {
     ]
     // All actions like network/* (starts with network)
     const DOMAINS = [
+      // '@@i18n/',
```

Comment:
What is this?

Reply:
We need to keep a list of all available domains to be able enable/disable them without copying from dev. tools console.